crates/evm/evm/src/executors/invariant/mod.rs

```diff
@@ -315,6 +320,7 @@ impl<'a> InvariantExecutor<'a> {
             setup_contracts,
             project_contracts,
             artifact_filters: ArtifactFilters::default(),
+            history_map: RefCell::new(vec![0u8; COVERAGE_MAP_SIZE]),
```

Comment:
The corpus should probably be ran on startup and populate this from `run_invariant_test`

Reply:
I think we can move this under `InvariantTestData`

https://github.com/foundry-rs/foundry/blob/8e2a4c4e4224f1286f6d1dfa3cba29c999062718/crates/evm/evm/src/executors/invariant/mod.rs#L122

which is part of `InvariantTest` execution data

https://github.com/foundry-rs/foundry/blob/8e2a4c4e4224f1286f6d1dfa3cba29c999062718/crates/evm/evm/src/executors/invariant/mod.rs#L151-L152

that is created / populated on `prepare_test`

https://github.com/foundry-rs/foundry/blob/8e2a4c4e4224f1286f6d1dfa3cba29c999062718/crates/evm/evm/src/executors/invariant/mod.rs#L509-L517

Reply:
Ig if the corpus can't be shared globally, this may make sense.